skeleton/docker-entrypoint.sh

```diff
@@ -163,6 +163,14 @@ elif  [[ "$1" == "run" ]]; then
   exec $sudo $VENVBIN/zconsole run etc/${CONF} "${@:2}"
 elif  [[ "$1" == "addzopeuser" ]]; then
   exec $sudo $VENVBIN/addzopeuser -c etc/${CONF} "${@:2}"
+elif  [[ "$1" == "pack" ]]; then
+  if [[ -v ZEO_ADDRESS ]]; then
+    exec $sudo $VENVBIN/zeopack -d "${2:-0}" ${ZEO_ADDRESS}
+  elif [[ -v RELSTORAGE_DSN ]]; then
+    exec $sudo $VENVBIN/zodbpack -d "${2:-0}" etc/relstorage_pack.conf
+  else
+    exec $sudo $VENVBIN/zconsole run etc/${CONF} /app/scripts/pack.py
+  fi
```

Comment:
It would be nice to support calling these commands with their normal arguments (i.e. pass along any arguments that are provided when invoking docker-entrypoint.sh)....I think the only exception is the conf file which makes sense to handle like you did here.

And, I would avoid setting `-d` here; it's confusing when scripts have different defaults depending on where/how they are called.

Reply:
I have rewritten this to pass all parameters from the second one to the script, likewise the `run` or `addzopeuser` commands.